akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -147,6 +147,20 @@ object ByteString {
 
     private[akka] def writeToOutputStream(os: ObjectOutputStream): Unit =
       toByteString1.writeToOutputStream(os)
+
+    override def copyToBuffer(buffer: ByteBuffer): Int =
+      writeToBuffer(buffer, offset = 0)
+
+    /** Specialized for internal use, writing multiple ByteString1C into the same ByteBuffer. */
+    def writeToBuffer(buffer: ByteBuffer, offset: Int): Int = {
```

Comment:
Has to be at least `private[akka]` (makes sense, I'll make it so), since it needs to be called from `ByteStrings`